packages/react-component-library/src/components/Timeline/__tests__/Timeline.test.tsx

```diff
@@ -386,12 +386,13 @@ describe('Timeline', () => {
         )
       })
 
-      it('should apply `aria-label` for row headers', () => {
+      it('adds text to the row headers', () => {
         const rowHeaders = wrapper.getAllByTestId('timeline-row-header')
 
-        expect(rowHeaders[0]).toHaveAttribute('aria-label', 'Months')
-        expect(rowHeaders[1]).toHaveAttribute('aria-label', 'Weeks')
-        expect(rowHeaders[2]).toHaveAttribute('aria-label', 'Days')
+        // Text is visually hidden
+        expect(rowHeaders[0]).toHaveTextContent('Months')
+        expect(rowHeaders[1]).toHaveTextContent('Weeks')
+        expect(rowHeaders[2]).toHaveTextContent('Days')
```

Comment:
Not really a nice way to check that the text is actually visually hidden here (there are the styles but they're an implementation detail) – the visual regression tests should cover the visibility of the text in any case.